### PR TITLE
completions: add zsh completion for riverctl

### PR DESCRIPTION
**Pull request by novakne**
I think I didn't forget any command, it works for all commands and some sub-commands for when there is a choice like `next previous`

### CODE REVIEW

File: completions/zsh/_riverctl [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+      |background-color\
+      |border-color-focused\
+      |border-color-unfocused\
+      |border-width\
+      |opacity\
+      |outer-padding\
+      |set-repeat\
+      |view-padding\
+      |xcursor-theme\
+      |outputs)
```
Comment (YerinAlexey) [inline completions/zsh/_riverctl:106]: I think this could be simplified with `_cmds`

Comment (novakne) [reply completions/zsh/_riverctl:106]: Oh yeah it works, didn't think about it, just
``` bash
  case "$words[1]" in
    _cmds) return 0 ;;
```

Comment (YerinAlexey) [reply completions/zsh/_riverctl:106]: > ```shell
>   case "$words[1]" in
>     _cmds) return 0 ;;
> ```

Wait, that's illegal

I would expect something like this:

```shell
case "$words[1]" in
  # snip
  *)
    return 0
  ;;
esac
```

Comment (novakne) [reply completions/zsh/_riverctl]: Yeah it's better

**Review by YerinAlexey — APPROVED** [review]
LGTM!

**Review by ifreund — APPROVED** [review]
Thanks!

**Pull request merged**